api/message.go

```diff
@@ -230,7 +234,7 @@ func (c *Client) SendEmbed(
 	channelID discord.ChannelID, e discord.Embed) (*discord.Message, error) {
```

Comment:
Would `SendEmbed(discord.ChannelID, ...discord.Embed)` be a better API for this?